### PR TITLE
Search Results Page Basic

### PR DESCRIPTION
**Pull request by MrMaksimize**
Hold Por favor

### CODE REVIEW

File: test/queryForm.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+
```
Comment (chrisrodz) [inline test/queryForm.js]: This file shouldn't be here?

**Comment by ainsleywagon** [comment]
Hey - it doesn't appear that the route is set up for the search page, therefore I cannot style :smile:.

**Comment by MrMaksimize** [comment]
The result page should be here :)

http://bizwallet-staging.herokuapp.com/results/all

I will probably end up switching the actual URL but it shouldn't affect the
style :)

Gracias Ainsley :)

On Friday, July 11, 2014, Ainsley notifications@github.com wrote:

> Hey - it doesn't appear that the route is set up for the search page,
> therefore I cannot style [image: :smile:].
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CoquiCoders/bizwallet/pull/25#issuecomment-48800570.

## 

Sorry for the short message, this was sent from my mobile device.

**Comment by ainsleywagon** [comment]
Thanks Max!
Have a great weekend.

On Fri, Jul 11, 2014 at 9:43 PM, Maksim Pecherskiy <notifications@github.com

> wrote:
> 
> The result page should be here :)
> 
> http://bizwallet-staging.herokuapp.com/results/all
> 
> I will probably end up switching the actual URL but it shouldn't affect
> the
> style :)
> 
> Gracias Ainsley :)
> 
> On Friday, July 11, 2014, Ainsley notifications@github.com wrote:
> 
> > Hey - it doesn't appear that the route is set up for the search page,
> > therefore I cannot style [image: :smile:].
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/CoquiCoders/bizwallet/pull/25#issuecomment-48800570.
> 
> ## 
> 
> Sorry for the short message, this was sent from my mobile device.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CoquiCoders/bizwallet/pull/25#issuecomment-48802287.

## 

Ainsley Wagoner | @ainsleywagon https://twitter.com/ainsleywagon
2014 Code for America Fellow
Team Puerto Rico | Follow our journey http://coquicoders.org/

**Comment by MrMaksimize** [comment]
De nada! Y tu tambien :)

On Saturday, July 12, 2014, Ainsley notifications@github.com wrote:

> Thanks Max!
> Have a great weekend.
> 
> On Fri, Jul 11, 2014 at 9:43 PM, Maksim Pecherskiy <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> 
> > wrote:
> > 
> > The result page should be here :)
> > 
> > http://bizwallet-staging.herokuapp.com/results/all
> > 
> > I will probably end up switching the actual URL but it shouldn't affect
> > the
> > style :)
> > 
> > Gracias Ainsley :)
> > 
> > On Friday, July 11, 2014, Ainsley <notifications@github.com
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> > 
> > > Hey - it doesn't appear that the route is set up for the search page,
> > > therefore I cannot style [image: :smile:].
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> > > https://github.com/CoquiCoders/bizwallet/pull/25#issuecomment-48800570>.
> > 
> > ## 
> > 
> > Sorry for the short message, this was sent from my mobile device.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/CoquiCoders/bizwallet/pull/25#issuecomment-48802287.
> 
> ## 
> 
> Ainsley Wagoner | @ainsleywagon https://twitter.com/ainsleywagon
> 2014 Code for America Fellow
> Team Puerto Rico | Follow our journey http://coquicoders.org/
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CoquiCoders/bizwallet/pull/25#issuecomment-48822396.

## 

Sorry for the short message, this was sent from my mobile device.

**Pull request merged**